docs/source/reference/migration/from-router-v1.mdx

```diff
@@ -35,84 +77,119 @@ Default values of some GraphOS reporting metrics have been changed from v1.x to
 
 ## Removed features in GraphOS Router v2.x
 
-### Support for Scaffold
+Follow the upgrade steps below for each feature of router v1.x that's been removed in router v2.x.
 
-<!-- PR: https://github.com/apollographql/router/pull/6274 -->
+### Removed busy timer for request processing duration
 
-In Router v1.x Scaffold could be used to generate boilerplate source code for a Rust Plugin which extended functionality in a custom Router. This facility has been removed.
+In `context::Context` that's typically used for router customizations, methods and structs related to request processing duration have been removed, because request processing duration is already included as part of spans sent by the
+router. Users customizing the router with Rhai scripts, Rust scripts, or coprocessors don't need to track this information manually. 
 
-Source code generated using Scaffold will continue to compile; so existing Rust plugins will be unaffected by this change.
+**Upgrade step**: remove calls and uses of the following methods and structs from `context::Context`:
```

Comment:
@goto-bus-stop - added a couple explicit **Upgrade step** examples. I'll continue populating and copyediting the sections for each item in your draft list